### PR TITLE
COMMERCE-11826 Automate tests for user story COMMERCE-11161

### PR DESCRIPTION
**Pull request by baharturk**
https://liferay.atlassian.net/browse/COMMERCE-11826

### CODE REVIEW

**Comment by baharturk** [comment]
ci:test:sf

**Comment by baharturk** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceAccountSupplierRole.testcase [hunk 1]
```diff
@@ -435,6 +597,89 @@ definition {
 		}
 	}
 
+	@description = "COMMERCE-11611. As a supplier account user, I cannot view the products in a catalog the account is not linked to."
+	@priority = 4
+	test CannotViewProductsInACatalogNotLinkedToSupplierAccount {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceAccountSupplierRole.testcase:602]:
```suggestion
	test CanSupplierViewOnlyLinkedCatalogProducts {
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceAccountSupplierRole.testcase [hunk 2]
```diff
@@ -435,6 +597,89 @@ definition {
...
+		}
+
+		task ("And also New Product from Master Catalog") {
+			CommerceNavigator.searchEntry(entryName = "New Product");
+
+			AssertElementNotPresent(
+				key_entryName = "New Product",
+				locator1 = "CommerceNavigation#ENTRY_NAME");
+		}
+	}
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceAccountSupplierRole.testcase:681]: Can you associate the supplier with the catalog you created and change your assertions accordingly? Thanks!

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by baharturk** [comment]
ci:test:sf

**Comment by baharturk** [comment]
ci:test:sf

**Comment by baharturk** [comment]
ci:test:commerce-functional

**Pull request closed without merging**